_projects/2022/100481925/100481925.Rmd

```diff
@@ -0,0 +1,1481 @@
+  </figcaption>
+  </p>
+</div>
+
+However, there is another important piece of information in Preethi Lodha's graph, which is the dollar amount spent by each generation for each expenditure category. This is shown within each point and is undoubtedly important if one wants to know how spending is distributed; however, that oscillation is not represented and can be misleading. This point will be discussed later.
+
+## What can be seen here? Encodings, channels and strengths
+There are 14 spending categories represented on the Y axis and 5 generations or age ranges showed on the X axis, above the graph and below the title and subtitle, from the Silent generation (the oldest) to the Z generation.
+
+According to Cleveland (1985), graphs encode data in distinct elements, such as symbols, colors, position, etc. When we study a graphic we perform several mental-visual tasks to extract quantitative information. Some of the basic judgements we perform to decode quantitative information are 1) angle, 2) area, 3) color hue, 4) color saturation, 5) density, 6) length, 7) position along a common scale, 8) position along identical, nonaligned scales, 9) slope, 10) volume (Cleveland, 1985).
+
+In consideration of this, the information in the bump chart can be extracted by judgements of position along the vertical scale, which represents the spending categories; the color of each line, which represents the categories of expenditure; the position of each point on the X-axis, which represents the generation; the thickness of the lines, which represents the money spent in each category (the thicker the line the more money has been spent). The closeness of the dots stands for the closeness of the ranking positions. The size of the points does not represent anything.
+
+Cleveland (1985) also discusses *detection*, which he qualifies as the most fundamental perception issue, because before executing any graphical perception task, we must be able to detect the graphical element. Lodha’s bump chart does not show overlaps in the dots, which together with color and position is the most important symbol. The jumps in the ranking are well understood.
+
+### Replication in R
+An almost exact replica was created using `ggplot2`.
+
+The libraries used were as follows:
+
+```{r}
+# Import libraries
+library(tidyverse) # for data manipulation
+library(ggplot2) # for data graphics, based on the Grammar of Graphics (Wilkinson 2005).
+library(ggtext) # provides markdown and HTML text rendering
+library(grid) # for adding the two white lines at the top and bottom
+```
+
+Then, the data was imported. As mentioned above, the data is only a pdf, so the values had to be transcribed manually. 
+
+```{r}
+df <- read_csv(
+  "https://github.com/carolinacornejocastellano/bump-chart/raw/main/data.csv")
```

Comment:
The same for the data, meaning: please download the dataset and put it here with the rest of the files, then reference that with a path relative to the Rmd file.